subgraph.yaml

```diff
@@ -150,3 +204,30 @@ dataSources:
         - event: Settled(uint256,uint256,bool,indexed address)
           handler: handleSettled
       file: ./src/market.ts
+
+  - kind: ethereum/contract
+    name: HL Mock DIA 3
```

Comment:
and here

Reply:
see replies to https://github.com/horse-link/horse.link.subgraph/pull/4#discussion_r1014939762

Reply:
@brettvniekerk it's not useful to have duplicate mocks, and can only cause confusion. Let's just have the one.